src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/ValueTaskAwaiter.cs

```diff
@@ -17,7 +17,9 @@ namespace System.Runtime.CompilerServices
     public readonly struct ValueTaskAwaiter : ICriticalNotifyCompletion, IStateMachineBoxAwareAwaiter
     {
         /// <summary>Shim used to invoke an <see cref="Action"/> passed as the state argument to a <see cref="Action{Object}"/>.</summary>
-        internal static readonly Action<object?> s_invokeActionDelegate = state =>
+        internal static readonly Action<object?> s_invokeActionDelegate = new Action<object?>(InvokeAction);
```

Comment:
Do we need the explicit `new Delegate` syntax, or can this (and the others similarly be):
```C#
internal static readonly Action<object?> s_invokeActionDelegate = InvokeAction;
```
?

Reply:
Yes, looks like that is valid